Dockerfile.buster

```diff
@@ -0,0 +1,94 @@
+ARG charge_lnd_tag=v0.2.8
```

Comment:
@AaronDewes To be honest, having two dockerfiles is kinda ugly and error prone, but I guess this is the way to go for now... Do you have any idea about how soon Umbrel will also move to Bullseye? Otherwise, I guess I'll look into a way to template the Dockerfile and extract out the common code.

Reply:
> Otherwise, I guess I'll look into a way to template the Dockerfile and extract out the common code.

I'm planning to try do that soon.